src/Raven.Studio/wwwroot/App/views/manage/threadStackTrace.html

```diff
@@ -0,0 +1,56 @@
+<div class="modal-dialog modal-lg" role="document" id="threadStackTraceDialog">
+    <div class="modal-content">
+        <div class="modal-header">
+            <button class="close" data-dismiss="modal" aria-label="Close"><i class="icon-cancel"></i></button>
```

Comment:
![image](https://user-images.githubusercontent.com/1250431/139745375-61e57392-b6a7-43fe-be81-ad18fb82d5a7.png)
can we use similar layout?

Reply:
done 
![image](https://user-images.githubusercontent.com/17376974/139800374-de2f7b06-84e1-41bc-a76f-bf43cd484ee1.png)